dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Maintain Kubernetes Nodes that Hold the TiDB Cluster
+summary: Learn how to maintain Kubernetes nodes that hold the TiDB cluster.
+category: how-to
+---
+
+# Maintain Kubernetes Nodes that Hold the TiDB Cluster
+
+TiDB is a highly available database that can run smoothly when some of the database nodes go offline. For this reason, you can safely shut down and maintain the Kubernetes nodes at the bottom layer without influencing TiDB's service. Specifically, you need to adopt various maintenance strategies when handling nodes that hold PD, TiKV, and TiDB instances because of their different features.
+
+This document introduces how to perform a temporary or long-term maintenance task for the Kubernetes nodes.
+
+## Prerequisites
+
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [`tkctl`](/tidb-in-kubernetes/tkctl.md)
```

Comment:
```suggestion
- [`tkctl`](/tidb-in-kubernetes/tkctl.md)
```
[`tkctl`](/tidb-in-kubernetes/tkctl.md) is pending transaltion by @CaitinChen. This PR should be merged after this link works.

Reply:
Got it.